includes/connect/class-wc-stripe-connect.php

```diff
@@ -265,5 +302,80 @@ private function record_account_connect_track_event( bool $had_error ) {
 			// a queue wouldn't be processed due to the redirect that comes after.
 			WC_Tracks::record_event( $event_name, [ 'is_test_mode' => $is_test ] );
 		}
+
+		/**
+		 * Schedules the App OAuth connection refresh.
+		 *
+		 * @since 8.6.0
+		 */
+		private function schedule_connection_refresh() {
+			if ( ! $this->is_connected_via_app_oauth() ) {
+				return;
+			}
+
+			/**
+			 * Filters the frequency with which the App OAuth connection should be refreshed.
+			 * Access tokens expire in 1 hour, and there seem to be no way to customize that from the Stripe Dashboard:
+			 * https://docs.stripe.com/stripe-apps/api-authentication/oauth#refresh-access-token
+			 * We schedule the connection refresh every 55 minutues.
+			 *
+			 * @param int $interval refresh interval
+			 *
+			 * @since 8.6.0
+			 */
+			$interval = apply_filters( 'wc_stripe_connection_refresh_interval', HOUR_IN_SECONDS - 5 * MINUTE_IN_SECONDS );
+
+			// Make sure that all refresh actions are cancelled before scheduling it.
+			$this->unschedule_connection_refresh();
+
+			as_schedule_single_action( time() + $interval, 'wc_stripe_refresh_connection', [], WC_Stripe_Action_Scheduler_Service::GROUP_ID, true, 0 );
```

Comment:
Oh I noticed you've already updated this to a priority 0. This feature was added in AS 3.6.0. I'll need to double check when WC updated to that version.

Reply:
They updated it to 3.6.1 in https://github.com/woocommerce/woocommerce/pull/38721 which was 7.9.0 of WooCommerce. Stripe requires WC 8.5 so I think we're all goo. 👍

Reply:
Actually our `WC_STRIPE_MIN_WC_VER` is set to 7.4. So we might not be good.

Reply:
You can use the changes I made to Subscriptions core here for inspiration. 

https://github.com/Automattic/woocommerce-subscriptions-core/pull/649/files#diff-8aefb8f2042be5275a619e3cc7fce879990e4c0511e0a70a8cd7fa1cec2d735bR265

Reply:
Actually, we should bump them all to align with our [support policy](https://developer.woocommerce.com/2021/08/26/developer-advisory-woocommerce-stripe-support-policy-change/)

Reply:
According to that policy, we need to support `WC 8.9, 9.0, and 9.1`, and `WP 6.4, 6.5, and 6.6`
WooCommerce 8.9 requires a minimum WP version of 6.4 so we don't need any exceptions there
For PHP, they both require 7.4... so that doesn't change.